src/Components/AppHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import Logo from '../Assets/Images/AppHeader/Logo.svg'
+import Bag from '../Assets/Images/AppHeader/Bag.svg'
+import Profile from '../Assets/Images/AppHeader/Profile.svg'
+import Search from '../Assets/Images/AppHeader/Search.svg'
+import Hamburger from '../Assets/Images/AppHeader/Hamburger.svg'
+export const AppHeader = () => {
+  return (
+    <div className='app-header flex justify-between items-center'>
+      <img src={Hamburger} alt="Hamburger" className='hamburger' />
+      <img src={Logo} alt="Logo" className="logo" />
+
+      <div className="right-nav flex">
+        <img src={Search} alt="Search" className='search'/>
+        <img src={Profile} alt="Profile" className='profile'/>
+        <img src={Bag} alt="Bug" />
+      </div>
+
+      <div className=""></div>
```

Comment:
why empty div?

Reply:
I used an empty div because, in the first version where I implemented "justify between," I encountered the following result:
Screenshot 2023-07-25 at 14 07 33 1

The problem here is that the center logo is not actually in the center. To solve this, I decided to set the right icons as "absolute" and created an empty div and that helped me to set my goal:
![Screenshot 2023-07-25 at 14 10 36](https://github.com/ShaharSaadon/epicure-frontend/assets/123458593/fc235a7d-8829-4b34-85a0-966ec7cffae2)